frontend/src/components/ProfileDropdown.vue

```diff
@@ -233,7 +232,7 @@ export default defineComponent({
         pushNotification({
           module: "bytebase",
           style: "SUCCESS",
-          title: info,
+          title: String(info),
```

Comment:
The type of "info" param is ServerInfo. Is it necessary to do type casting here? Or it would be better declaring info param as "string" type.